libs/ddhub-client-gateway-env/src/lib/open-telemetry.env.ts

```diff
@@ -0,0 +1,67 @@
+import * as Joi from 'joi';
+
+export const OPEN_TELEMETRY_ENVS = Joi.object({
+  OPENTELEMETRY_ENABLED: Joi.boolean().default(false),
+  OTEL_IGNORED_ROUTES: Joi.alternatives()
+    .conditional('OPENTELEMETRY_ENABLED', {
+      is: true,
+      then: Joi.string().default('health,api/v2/health').required(),
```

Comment:
it will fail when OPENTELEMETRY_ENABLED is false and no `otherwise` condition

Reply:
already fixed here https://github.com/energywebfoundation/ddhub-client-gateway/pull/16